src/js/components/Calendar/Calendar.js

```diff
@@ -847,21 +847,24 @@ const Calendar = forwardRef(
         if (!children) {
           days.push(
             <CalendarDay
+              activeProp={active}
+              day={day}
               key={day.getTime()}
+              disabledProp={dayDisabled && !!dayDisabled}
               buttonProps={{
                 a11yTitle: day.toDateString(),
-                active: active && active.getTime() === day.getTime(),
-                disabled: dayDisabled && !!dayDisabled,
-                onClick: () => onClick(dateObject),
-                onMouseOver: () => setActive(dateObject),
-                onMouseOut: () => setActive(undefined),
+                onClick:
+                  dayDisabled && !!dayDisabled
+                    ? () => {}
+                    : () => onClick(dateObject),
               }}
               isInRange={inRange}
               isSelected={selected}
               otherMonth={day.getMonth() !== reference.getMonth()}
               rangePosition={rangePosition}
               size={size}
               fill={fill}
+              ref={focusableDateRef}
```

Comment:
To make sure I'm understanding, we track this as a ref so we can programmatically focus the "active" day? If so, how come the `tabIndex={0}` can't just take care of that?

Reply:
> To make sure I'm understanding, we track this as a ref so we can programmatically focus the "active" day?
Correct

> If so, how come the tabIndex={0} can't just take care of that?
Setting tabIndex to 0 allows something to receive focus but adding/removing `tabIndex={0}` doesn't automatically change where the focus is placed

Reply:
I supposed my question is why we need to programmatically focus a day at all?

Reply:
Discussed on a call, just summarizing here for other people. Because the roving tab index pattern moves focus via arrow keys instead of the tab button, focus does need to be handled programmatically